lib/refresh.js

```diff
@@ -37,6 +52,12 @@ AuthTokenRefresh.use = function (name, strategy) {
     throw new Error('Cannot register: not an OAuth2 strategy');
   }
 
+  if (agent && typeof agent === 'string') {
```

Comment:
Consider maybe to change the condition to
` if (agent && typeof agent !== 'object') {`